server/opts.go

```diff
@@ -190,6 +190,8 @@ type Options struct {
 	WriteDeadline    time.Duration `json:"-"`
 	MaxClosedClients int           `json:"-"`
 	LameDuckDuration time.Duration `json:"-"`
+	// MaxTracedMessagePrintableLen is the maximum printable length for traced messages.
+	MaxTracedMessagePrintableLen int `json:"max_traced_message_printable_len"`
```

Comment:
Prefer shorter names here.

Reply:
Any special concerns? I think it is more clear.

Reply:
MaxTracedMsgLen